consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseDataAccessRequestAPI.java

```diff
@@ -603,4 +605,32 @@ private Document processDataSet(Document dataAccessRequest, ConsentDataSet conse
         return dataAccess;
     }
 
+    protected List<ResearcherProperty> updateResearcherIdentification(Document dataAccessRequest) {
+        Integer userId = dataAccessRequest.getInteger(DarConstants.USER_ID);
+        String linkedIn = dataAccessRequest.getString(ResearcherFields.LINKEDIN_PROFILE.getValue());
+        String orcId = dataAccessRequest.getString(ResearcherFields.ORCID.getValue());
+        String researcherGate = dataAccessRequest.getString(ResearcherFields.RESEARCHER_GATE.getValue());
+        List<ResearcherProperty> rpList = new ArrayList<>();
+        if (!Objects.equals(linkedIn, researcherPropertyDAO.findPropertyValueByPK(userId, ResearcherFields.LINKEDIN_PROFILE.getValue()))
+                || !Objects.equals(orcId, researcherPropertyDAO.findPropertyValueByPK(userId, ResearcherFields.ORCID.getValue()))
+                || !Objects.equals(researcherGate, researcherPropertyDAO.findPropertyValueByPK(userId, ResearcherFields.RESEARCHER_GATE.getValue()))) {
```

Comment:
This if check has 3 internal `researcherPropertyDAO.findPropertyValueByPK` queries which increases the minimum number you're making. I'm suggesting that you don't need to make those queries at all (and therefore, the if check too).

Pseudocode:
```
delete all user properties of type [linkedIn, orcId, researcherGate]
if (linkedIn) add new linkedIn property
if (orcId) add new orcId property
if (researcherGate) add new researcherGate property
```